### DISCUSSION TITLE
纹理背景设置textureFillMode为2不显示背景图，0和1都显示

### DISCUSSION THREAD

**Discussion by fhwkfu** [galacean/discussion#96] [Engine]
// 纹理背景
    background.mode = BackgroundMode.Texture
    background.textureFillMode = 2 // BackgroundTextureFillMode.FitWidth:0 FitHeight:1 Fill:2 
    const texture = await engine.resourceManager.load({
      url: 'https://gw.alipayobjects.com/mdn/rms_2e421e/afts/img/A*BcWiRYM7hroAAAAAAAAAAAAAARQnAQ',
      type: AssetType.Texture2D
    })
    background.texture = texture

**Comment by cptbtptpbcptdtptp** [comment]
引擎的版本号是多少，运行的环境是什么，还有建议输出一下 texture，有可能是纹理没加载成功导致的。
